evaluator/src/main/kotlin/com/xebia/functional/xef/evaluator/EitherOps.kt

```diff
@@ -0,0 +1,24 @@
+package com.xebia.functional.xef.evaluator
+
+import arrow.core.Either
+import arrow.core.NonEmptyList
+import com.xebia.functional.xef.evaluator.models.SuiteSpec
+import com.xebia.functional.xef.evaluator.models.errors.ValidationError
+import java.io.File
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+object EitherOps {
```

Comment:
Question, in Kotlin is this a good practice (nested the functions in an object)? I have seen in the project the extension functions defined at top level